### ISSUE TITLE
Conan source warning: This package defines both 'os' and 'os_build'

### ISSUE THREAD

**Issue by Israphel** [conan-io/conan#2368]
Conan version: **1.0.3**
Recipe settings:
`settings = "os", "compiler", "build_type", "arch"`

Profile:
```
[settings]
os=Linux
os.distro=rhel7
arch=x86_64
compiler=gcc
compiler.version=4.8
compiler.libcxx=libstdc++
build_type=Debug
arch_build=x86_64
os_build=Linux
```

When running **conan source .** I got:
```
PROJECT: ************************************************************
PROJECT:   This package defines both 'os' and 'os_build' 
PROJECT:   Please use 'os' for libraries and 'os_build'
PROJECT:   only for build-requires used for cross-building
PROJECT: ************************************************************
```

When removing **os_build** from the profile, the warning is gone.

**Comment by memsharded** [comment]
Yes, this is totally intended. What is your use case that requires defining both of them?

**Comment by memsharded** [comment]
Sorry, I am not able to reproduce this. I guess that you have defined that in your default profile, did you?

Could you please share your recipe?

**Comment by memsharded** [comment]
Now I have been able to see it, but not clear what the difference was. Investigating.

**Comment by memsharded** [comment]
Yes, this is a small bug, that warning slipped to the wrong place.
Just disregard it. We will be fixing it for next 1.1, thanks very much for reporting.

**Comment by memsharded** [comment]
Fix released in 1.0.4, please check.
